src/Explorer/Explorer.ts

```diff
@@ -509,6 +510,14 @@ export default class Explorer {
       return false;
     });
 
+    this.isServerlessEnabled = ko.computed(
+      () =>
+        this.databaseAccount &&
+        this.databaseAccount()?.properties?.capabilities?.find(
+          item => item.name === Constants.CapabilityNames.EnableServerless
+        ) !== undefined

```

Comment:
you dont need "!==undefined"

Reply:
It is not able to convert implicitly to `boolean` according to the linter so need an explicit check (or conversion).